LICENSE.md

```diff
@@ -1,6 +1,8 @@
 All **source code** is copyright © 2019-2021 [VCV](https://vcvrack.com/) and contributers.
 
-This program is free software: you can redistribute it and/or modify it under the terms of the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.en.html) as published by the [Free Software Foundation](https://www.fsf.org/), either version 3 of the License, or (at your option) any later version.
+This program is free software: you can redistribute it and/or modify it under the terms of the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.en.html) as published by the [Free Software Foundation](https://www.fsf.org/), version 3 of the License (GPL-3.0-only).
```

Comment:
change to `GPL-3.0-only` to be compatible with Noise Plethora CC-BY-SA v4 license used for plugins.

Reply:
See: https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses

Also last paragraph of https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses

Basically, GPL version can be increased at a later date if determined compatible with CC, but for now GPLv3 only.

If I've misunderstood then let me know!

Reply:
That doesnt invalidate the code "or later" clause as I see it.
And that whole compatible license thing is for when generating modifications of the artwork files, say someone mixing *artwork* that is CC-BY with *artwork* that is GPL.

So as I see it, the usage within Rack modules does not apply, as the separation between artwork and code is quite clear.
Otherwise one would not be able to use CC-NC or CC-ND like Rack itself or Fundamental does, as those licenses are inherently incompatible with GPL.

Reply:
Just to clarify, I am adapting code (the code for each of the 30 Noise Plethora plugin) not artwork in this instance. The relevant lines (emphasis mine) are:

> You must specify version 3 of the GPL when redistributing a GPL project into which BY-SA 4.0 content is adapted.

> Because this compatibility determination is **limited to version 3 of the GPL**, reusers that adapt BY-SA 4.0 content and incorporate it into GPL-licensed projects are only able to use GPLv3 unless and until other GPL versions are declared compatible.

Reply:
oh so the actual code this is based on is CC licensed? that is very odd choice. but then would make sense yes

Reply:
exactly:

license text (last sentence): https://github.com/Befaco/Noise_plethora/blob/master/README.md
example plugin: https://github.com/Befaco/Noise_plethora/blob/master/P_TriFMcluster.hpp

Reply:
But this license change would be only for this specific module, then?
Doesn't make a lot of sense to re-license the entire project like this.

Reply:
all the code is merged together at the end as single binary, so the most restrictive of licenses will apply.
if there is gplv3+ and gplv3-only on the same project, final binary is gplv3-only

Reply:
True-true, wasn't paying attention. Still not sure if it makes sense to merge mixed licensed stuff like this and in essence "downgrade" the existing code.

Reply:
Team Befaco have [relicensed](https://github.com/Befaco/Noise_Plethora) the plugin files to GPL-v3-or-later, so we can go back to GPL-v3-or-later. This PR should now reflect the updated license.